pkg/distributor/validate_test.go

```diff
@@ -1127,3 +1215,9 @@ func TestValidUTF8Message(t *testing.T) {
 		}
 	}
 }
+
+func TestLabelValueHashLen(t *testing.T) {
+	x := strings.Repeat("x", 100)
+	x = hashLabelValueInto(x, x)
+	require.Len(t, x, validation.LabelValueHashLen)
+}
```

Comment:
The original test doesn't make much sense to me. BTW, why do the hexadecimal representations have to start with 0?
```suggestion
func TestLabelValueHashLen(t *testing.T) {
func TestHashLabelValueInto(t *testing.T) {
	x := strings.Repeat("x", 100)
	var err error
	x, err = hashLabelValueInto(x[:validation.LabelValueHashLen], x)
	require.NoError(t, err)
	require.Equal(t, "(hash:052c9e7cec411035", x)
}
```